### PR TITLE
IPC / Machine People Species

### PR DESCRIPTION
**Pull request by Peptide90**
## About the PR

Port of IPC / Machine People from Ekrixi. Will need some tweaking and improvements over time to fit out gameplay but this is a start.

- [ ] Fix their examine description.
- [ ] Test roundstart species code to make sure marking etc all work.
- [ ] See if we want them to "breathe".
- [ ] Fix footstep sounds if possible, needs to be more metallic when barefoot.
- [ ] Test talking sounds.
- [ ] Add some sort of robot scream sound.

## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### CODE REVIEW

**Comment by UbaserB** [comment]
based

**Comment by K-Dynamic** [comment]
Letsa fucking gooo

Let them heal by eating trash and lightbulbs

**Comment by DEATHB4DEFEAT** [comment]
https://github.com/Simple-Station/Parkstation/pull/97

**Comment by BasedUser** [comment]
i beg you use the sirena IPCs they're literally just ekrixi IPCs except i did more than just add emote sounds to them (blood changes, zombification, REINHERITING SO THEY ARENT ORGANICS)

**Comment by BasedUser** [comment]
also your commits are fucked its not just IPCs youre porting over

**Comment by Just-a-Unity-Dev** [comment]
hi! i wrote IPCs for ekrixi, i would just like to say you are better off making it yourself rather than porting it, my implementation for IPCs were rushed and pretty meh overall, plus i dont think they were ever touched after species rework. so yea

**Comment by lzk228** [comment]
MACHINE PEOPLE LOL
call it IPC and nothing else

**Comment by Peptide90** [comment]
Whoops lol, pushed this to the wrong repo. Sorry to dissapoint folks but I'm not sure I'll be upstreaming this unless upstream actually want it. Sounds like a bigger job that I'd want...

**Comment by Peptide90** [comment]
> [Simple-Station/Parkstation#97](https://github.com/Simple-Station/Parkstation/pull/97)

Also never knew parkstation had IPCs, neat. Might have to look at that instead of the Sirena ones I was about to use.

File: README.md
[the remark this answers is not in the document]
Comment (Peptide90) [reply README.md:1]: Yeah I must have made this branch on the wrong repo. My bad.

**Comment by VasilisThePikachu** [comment]
Just a notice if you reopen this. New species are restricted in #24346 

You will need a design doc

**Comment by Peptide90** [comment]
> Just a notice if you reopen this. New species are restricted in #24346
> 
> You will need a [design doc](https://docs.spacestation14.com/en/templates/proposal.html) and will be scrutinized heavily

Yep, exactly why it's not getting opened :)

**Pull request closed without merging**